### DISCUSSION TITLE
PS2, Gamecube, Wii cores?

### DISCUSSION THREAD

**Discussion by olive1234567890** [forkymcforkface/RGBPi-Extra#4] [Ideas]
Since pi5 runs these systems well in general, is it possible to add these? thank you

**Comment by forkymcforkface** [comment]
The pi5 can't do interlacing through the gpio. Any games that look ok with 240p on these systems?

**Comment by olive1234567890** [comment]
I have played forced 240p with Swiss on many GameCube games and I felt it
was okay

**Comment by forkymcforkface** [comment]
Gamecube/Wii core added for Pi5 rgbpi-extra
